.travis.yml

```diff
@@ -93,7 +93,7 @@ before_install:
   # Keep track of which version of node we're running the specs against
   - bin/git-submodule-fast-install
   - node -v
-  - npm install jshint uglify-js
+  - npm install
```

Comment:
We should probably update the [HACKING.md](https://github.com/opal/opal/blob/master/HACKING.md) file:

```diff
- $ npm install -g jshint
+ $ npm install
```

Reply:
Thanks, moved everything to `bin/setup`, both for HACKING and in the README